src/systems/websocket_server/WebsocketServer.hh

```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+*
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef GZ_SIM_SYSTEMS_WEBSOCKETSERVER_HH_
```

Comment:
note about where this file is coming from

Reply:
added note in [2c75261](https://github.com/gazebosim/gz-sim/pull/2792/commits/2c75261bbbd2181cc71682053ad976b11f85c650)